README.md

```diff
@@ -322,18 +322,43 @@ Another feature of RapydScript is ability to have functions as part of your obje
 params = {
 	width:	50,
 	height:	30,
-	onclick:	def(event):
-		alert("you clicked me"),
-	onmouseover:	def(event):
+	onclick: (def(event):
+		alert("you clicked me")
+	),
+	onmouseover: (def(event):
 		$(this).css('background', 'red')
-	,
-	onmouseout:	def(event):
+	),
+	onmouseout: (def(event):
 		# reset the background
 		$(this).css('background', '')
+	)
 }
 ```
 
-Note the comma on a new line following a function declaration, it needs to be there to let the compiler know there are more attributes in this object literal, yet it can't go on the same line as the function since it would get parsed as part of the function block. Like Python, however, RapydScript supports new-line shorthand using a `;`, which you could use to place the comma on the same line:
+<details>
+	<summary>Optional: You can surround inline functions with parentheses</summary>
+
+```
```

Comment:
@hangtwenty 
Hi!
I guess here should be an example of inline function definition using parentheses, no?